### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by nilab-khwaja**
- Add any missing JSX elements to the project (without adding extra functionality)
-  final design (this includes all elements + the same styling)

### CODE REVIEW

File: src/components/styles/Navbar.module.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  width: 170px;
+}
+
+.nav > div:nth-child(1) {
+  display: flex;
+  align-items: center;
+}
```
Comment (ademibowale) [inline src/components/styles/Navbar.module.css]: - [ ] I would suggest you to stick to the ziplin design as much as posible to make your app look professional ,kindly make the text Bookstor align with the page ,you can reduce the margin px to fit the design  as show on the design template . 
- [ ] Your design
Screenshot 2023-09-28 191307

- [ ] Given design
bookstore

File: src/components/styles/Book.module.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  display: flex;
+  list-style-type: none;
+  justify-content: space-between;
+  margin-top: 20px;
+}
+
+.book > div:nth-child(1) ul li button {
+  border: none;
+  background-color: white;
+}
```
Comment (ademibowale) [inline src/components/styles/Book.module.css:47]: - [ ] I would suggest you to make sure the Icon menu to be align with the addbook button as shown on the given design as you can see below your design is different to the original design please kindly stick to the given design as much as possible for user friendly.
- [ ] given design
bookstore

- [ ] your design
Screenshot 2023-09-28 191307

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @nilab-khwaja , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

- Descriptive Pull Request title and summary. ✔️
- All check passed✔️
- Good use of Github flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @nilab-khwaja👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Implemented all changes requested by the previous reviewer✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**